packages/fiori-freestyle-writer/src/index.ts

```diff
@@ -92,6 +104,12 @@ async function generate<T>(basePath: string, data: FreestyleApp<T>, fs?: Editor)
         fs.write(ui5LocalConfigPath, ui5LocalConfig.toString());
     }
 
+    // Entend ui5-local.yaml with additional UI5 lib
```

Comment:
typo `Entend`

Reply:
Addressed in [89f5a8a](https://github.com/SAP/open-ux-tools/pull/1948/commits/89f5a8a9cfc05c49dff465ee079a7226c8a0b1a4), thanks!